src/io/windows.zig

```diff
@@ -977,31 +981,37 @@ pub const IO = struct {
         const shared_mode: os.windows.DWORD = 0;
 
         // O_RDWR
+        // Zig's mask seems wonky; according to
+        // https://learn.microsoft.com/en-us/windows/win32/api/winternl/nf-winternl-ntcreatefile
+        // FILE_GENERIC_READ should include SYNCHRONIZE but it does not.
```

Comment:
Could you file an upstream issue here?

Reply:
Dug a bit further, it's not an upstream issue, but rather confusing naming.

`GENERIC_READ`, which you can use for files too, is different from `FILE_GENERIC_READ` which includes `SYNCHRONIZE`.

I'll put up a small PR to update the comment.